src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
+    }
```

Comment:
```suggestion
public class ColorSupplier {
	Random random = new Random();
	
    public String getRandomColor() {
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].toString();
    }
```

давай вынесем переменную Random на уровень класса, что б не создавать каждый раз новый экземпляр, когда мы вызываем метод getRandomColor()